css/ryht-legislative-style.css

```diff
@@ -352,16 +352,32 @@ p.moreinfo  {
   font-family: "Montserrat", Arial, Helvetica, sans-serif;
   font-size:14px;
  /* width:auto !important;*/
-  }
+}
 
-  p.programs  {
+/* I'm starting this as a copy of .moreinfo, with just the top & bottom padding removed */
```

Comment:
I added a class for the new UI piece that lets the viewer switch between House and Senate districts.  Styling is currently based on `p.moreinfo` but if you want to tweak it, here's where to look.

Reply:
This is good, the styling is a nice bit of symmetry with the bottom of the flyout